spec/factories/seeds/school_factory.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory(:seed_school, class: "School") do
+    transient { location { Faker::Address.city } }
+    transient { domain { [location.parameterize, Faker::Internet.domain_suffix].join } }
+
+    urn { Faker::Number.unique.decimal_part(digits: 7).to_s }
+    name { [location, %w[High Grammar Infant Nursery].sample, "School"].join(" ") }
+    postcode { Faker::Address.postcode }
+    address_line1 { Faker::Address.street_address }
+    address_line3 { location }
+    domains { Array.wrap(domain) }
+    school_website { "www.#{domain}" }
+    primary_contact_email { Faker::Internet.email(domain:) }
```

Comment:
Don't think these are required to create a valid school

Reply:
I added these so the school pages looked fuller (and hopefully more realistic) in seeded environments. Think they'll be problematic?